ertviz/controllers/ensemble_selector_controller.py

```diff
@@ -58,7 +58,10 @@ def update_ensemble_selector(selected_ensembles, _, elements):
         ctx = dash.callback_context
         triggered_id = ctx.triggered[0]["prop_id"].split(".")[0]
 
-        if triggered_id == parent.uuid("ensemble-selection-store"):
+        if (
```

Comment:
Could you, maybe, make the function name more descriptive for `update_ensemble_selector`? Function `update_ensemble_selection` is fine!

Reply:
renamed to `update_ensemble_selector_graph`